### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by Leon30217** [flybywiresim/aircraft#2973] [Help] [answered]
Hello,
please apologize, but I do not know, where I can report suggestions for improvement or bugs?
Can you help me?

regards
Leon

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
If it’s an unknown issue or unrecognized request go to the issues tab on this github, if it’s just a discussion join our discord server https://discord.gg/flybywire

**Comment by hotshotp** [comment]
on the issues section of this github
